site/content/en/blog/releases/1.23.0-rc.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "1.23.0 - Release Candidate"
+linkTitle: "1.23.0-rc"
+date: "2022-05-03"
+---
+This is the 1.23.0-rc release of Agones.
+
+This release brings us support for Kubernetes 1.22, so if you are installing this release,  Please remember to upgrade your clusters.
+
+This release has lots of important bug fixes includes:
+
+* Fix for health controller race condition marking healthy pods as unhealthy.
+* Check for DeletionTimestamp of fleet and gameserverset before scaling.
+* GameServer stucks in Shutdown state preventing a rolling update to complete.
+*  Fix for GameServer back to Ready always.
+* Queue updated GameServers with DeletionTimestamp.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.23.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+- Upgrade to Kubernetes 1.22 [\#2494](https://github.com/googleforgames/agones/issues/2494)
+- Update golang version used in App Engine  [\#2382](https://github.com/googleforgames/agones/issues/2382)
+- Allocator controller arm64 img [\#2565](https://github.com/googleforgames/agones/pull/2565) ([Ludea](https://github.com/Ludea))
+- Add the new gke-gcloud-auth-plugin binary to the build image [\#2561](https://github.com/googleforgames/agones/pull/2561) ([roberthbailey](https://github.com/roberthbailey))
```

Comment:
This isn't a user facing enhancement, just a fix for our build system. I've adjusted the tag on the PR.